### PR TITLE
set file-size upper limit on option page

### PR DESCRIPTION
**Pull request by pastak**
[Screenshot from Gyazo](https://gyazo.com/92963aede52de97cdff22ecad476a638)

### CODE REVIEW

**Comment by pastak** [comment]
Update display [Screenshot from Gyazo](https://gyazo.com/2ce5e440139104cd01862120a6db06a9)

**Comment by pastak** [comment]
updated

[Screenshot from Gyazo](https://gyazo.com/0dddb2a5e682a059f038f206fd8e402b)

**Comment by rakusai** [comment]
- Why Japanese / English is mixed in the UI?

File: src/statics/_locales/en/messages.json [hunk 1]
```diff
@@ -46,5 +46,8 @@
   },
   "captureParentTitle": {
     "message": "Capture"
+  },
+  "fileSizeLimitText": {
+    "message": "Upper limit file size to upload(file whose size is larger will be compressed)"
```
Comment (rakusai) [inline src/statics/_locales/en/messages.json]: Maximum file size to upload (Files larger than this size will be compressed)

**Comment by pastak** [comment]
[Screenshot from Gyazo](https://gyazo.com/499d9f03e9c0f6a58e1ef21a9437a920)

File: src/statics/_locales/en/messages.json [hunk 2]
```diff
@@ -38,16 +38,31 @@
...
+    "message": "Maximum file size to upload"
+  },
+  "fileSizeLimitHelpText": {
+    "message": "Files larger than this size will be compressed"
+  },
+  "defaultActionLabel": {
+    "message": "Default action"
+  },
+  "pageScrollDelayLabel": {
+    "message": "Page scroll delay"
```
Comment (rakusai) [inline src/statics/_locales/en/messages.json]: Page scroll speed

File: src/statics/_locales/en/messages.json [hunk 3]
```diff
@@ -38,16 +38,31 @@
...
+  },
+  "pageScrollDelayHelpText": {
+    "message": "Please set slow if capture is broken"
+  },
+  "pageScrollDelayWords": {
+    "message": "Short,Normal,Long"
```
Comment (rakusai) [inline src/statics/_locales/en/messages.json]: Fast, Normal, Slow

File: src/statics/_locales/en/messages.json [hunk 4]
```diff
@@ -38,16 +38,31 @@
...
+    "message": "Files larger than this size will be compressed"
+  },
+  "defaultActionLabel": {
+    "message": "Default action"
+  },
+  "pageScrollDelayLabel": {
+    "message": "Page scroll delay"
+  },
+  "pageScrollDelayHelpText": {
+    "message": "Please set slow if capture is broken"
```
Comment (rakusai) [inline src/statics/_locales/en/messages.json]: Please set slow if images are broken

File: src/statics/option/option.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  width: 600px;
+  margin: 0;
+  padding: 0;
+}
+
+section {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #D7D7D7;
+  padding: 15px 10px 15px 15px;
```
Comment (rakusai) [inline src/statics/option/option.css]: padding: 15px;

**Comment by rakusai** [comment]
Looks Good! (Please fix English)

**Comment by pastak** [comment]
I updated English

**Pull request merged**